vmdb/app/helpers/provider_foreman_helper.rb

```diff
@@ -1,30 +1,126 @@
 module ProviderForemanHelper
   def textual_group_properties
-    items = %w(hostname configuration_profile configuration_profile_desc provider provider_url zone)
+    items = %w(hostname ipmi_present ipaddress mac_address configuration_profile_desc provider_name zone)
     items.collect { |m| send("textual_#{m}") }.flatten.compact
   end
 
   def textual_hostname
-    {:label => "#{title_for_host} Name", :value => @record.hostname}
+    h = {}
+    h[:label] = "#{title_for_host} Name"
+    h[:image] = "configured_system"
+    h[:value] = @record.hostname
+    h
   end
 
-  def textual_configuration_profile
-    {:label => "Configuration Profile", :value => @record.configuration_profile.try(:name)}
+  def textual_ipmi_present
+    {:label => "IPMI Present", :value => @record.ipmi_present}
```

Comment:
are we going to use `_("IPMI Present")`?
disclaimer: I'm not sure if that will work and if we want to do it.